qcodes/instrument_drivers/tektronix/Keithley_2450.py

```diff
@@ -109,6 +131,22 @@ def clear_trace(self) -> None:
         """
         self.write(":TRACe:CLEar")
 
+    def get_user_number(self) -> int:
+        return self._user_number
+
+    def set_user_number(self, value) -> None:
+        self._user_number = value
+
+    def get_user_delay(self) -> str:
+        get_cmd = f":SENSe:{self._proper_function}:DELay:USER" \
+                  f"{self.user_number()}?"
+        return self.ask(get_cmd)
+
+    def set_user_delay(self, value) -> None:
```

Comment:
do these methods need to be public? if this dfunctionality is supposed to be accessed via parameters, then please make these methods private .

Reply:
Good point. User should only call the parameters. I have set them to private.